getting-started/templates/systemlink-values.yaml

```diff
@@ -639,6 +650,31 @@ dataframeservice:
       ## Size of the queue for concurrent requests. If a request arrives to a pod with a full queue,
       ## the replica will return a 429 Error code.
       queueSize: 0
+    ## Configuration for the per-user rate limiter for metadata queries, including all of the following routes:
```

Comment:
'Configuration for the per-user rate limiter for metadata queries. This configuration includes all of the following routes:'